packages/indexer-generator/README.md

```diff
@@ -167,7 +166,7 @@ Get aggregated accesses by signer and month:
 Get the latest 1000 processed instructions:
 ```graphql
 {
-  events(account: "ELMTgR1fLdJeENJKWGWz3eCH8URpaSuPfCyAaphELTVJ", types: OrderUnstake, limit: 10) {
+  events(account: "ELMTgR1fLdJeENJKWGWz3eCH8URpaSuPfCyAaphELTVJ", types: OrderUnstake, limit: 1000) {
```

Comment:
Let's not make the user do something potentially dangerous directly at the beginning, it should've been 10 ^^